hack/generator/pkg/jsonast/testdata/OneOf/OneOf_generates_wrapper_type_with_marshal_helper.golden

```diff
@@ -27,8 +27,11 @@ type Foo struct {
 
 /*Generated from: https://test.test/schemas/2020-01-01/test.json*/
 type Test struct {
+	/* Bar: mutually exclusive with all other properties */
 	Bar *Bar `json:"bar"`
+	/* Baz: mutually exclusive with all other properties */
 	Baz *Baz `json:"baz"`
+	/* Foo: mutually exclusive with all other properties */
```

Comment:
The other comments lack whitespace around the value, because of limitations in CRD generation, do these need to follow suit? e.g. `/*Bar: mutually exclusive with all other properties*/`

Reply:
Fixed